IdentityCore/tests/automation/ui_tests_lib/lab_api/MSIDTestAutomationAppConfigurationRequest.m

```diff
@@ -74,6 +74,12 @@ - (nonnull id)copyWithZone:(nullable NSZone *)zone
 
 - (NSString *)requestOperationPath
 {
+    //If appID is present, append it to path to ensure querying specific apps results in going to the correct path
+    if (self.appId)
+    {
+        return [NSString stringWithFormat:@"%@/%@", @"app/", self.appId];
```

Comment:
Can be simplified?

return [NSString stringWithFormat:@"app/%@", self.appId];

Reply:
+1